Extension/package.nls.json

```diff
@@ -951,6 +951,33 @@
     "c_cpp.debuggers.vsCodeCommand.description": "VS Code command to be invoked. Can be a command in VS Code or an active extension.",
     "c_cpp.debuggers.vsCodeCommand.command.description": "VS Code command to be invoked.",
     "c_cpp.debuggers.vsCodeCommand.args.description": "Arguments to the VS Code command.",
+    "c_cpp.debuggers.cpplldb.debuggerPath.description": "Full path to the LLDB-DAP debugger executable. If not specified, the extension will search for the debugger in the system PATH.",
+    "c_cpp.debuggers.cpplldb.debuggerArgs.description": "Additional arguments for the LLDB-DAP debugger.",
+    "c_cpp.debuggers.cpplldb.disableaslr.description": "Disable Address Space Layout Randomization (if supported).",
```

Comment:
Should this be disableASLR?

Reply:
Oh, it's just the label of the string so I'm not sure if it matters?

Reply:
It's just a label. I cap'd the A